client/src/components/navBar.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+//navigation bar with list of restaurants 
+function Nav(props) {
+  return (
+    <nav>
+      <ul>
+        <li>Vermont-Tap-House</li>
+        <li>Panera-bread</li>
+        <li>longHorn-steakhouse</li>
+        <li>Chilis-grill-bar</li>
+        <li>Moes-southwest-grill</li>
+        <li>Bliss-bee</li>
+        <li>Texas-RoadHouse</li>
+        <li>The-Scale</li>
+      </ul>
+    </nav>
+  );
```

Comment:
Good start to this. Next try building it out into clickable links that bring you to the specific restaurant page by using a Restaurant component